src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs

```diff
@@ -12134,6 +12135,88 @@ public void StrongNameProviderWithCustomTempPath()
             Assert.True(!comp.SignUsingBuilder);
         }
 
+        [Theory]
+        [InlineData(@"/features:InterceptorsPreviewNamespaces=NS1.NS2;NS3.NS4")]
+        [InlineData(@"/features:""InterceptorsPreviewNamespaces=NS1.NS2;NS3.NS4""")]
```

Comment:
>NS1.NS2;NS3.NS4 [](http://example.com/codeflow?start=64&length=15)

Do we need to handle spaces? `= NS1; NS2"

Reply:
I'm not planning to currently. I'll add tests.